arch/arm/configs/miyoo_defconfig

```diff
@@ -126,10 +120,6 @@ CONFIG_USB_G_HID=m
 CONFIG_MMC=y
 CONFIG_MMC_SUNXI=y
 CONFIG_STAGING=y
-CONFIG_STAGING_MEDIA=y
-CONFIG_VIDEO_SUNXI=y
```

Comment:
why disable VIDEO_SUNXI and CMA afterall?

Reply:
It's not needed for uClibc, and since separate configs are required for musl/uClibc, the best option would be to move the defconfigs to Buildroot, which will build the kernel with a different defconfig depending on the selected libc.